tests/integration/ingestor-error-handling.test.js

```
@@ -0,0 +1,682 @@
+        const scenario = payload.scenario;
+        attemptCounts[scenario] = (attemptCounts[scenario] || 0) + 1;
+
+        switch (scenario) {
+          case 'success':
+            return [fixtures.retryComments[0]];
+            
+          case 'transient':
+            throw new Error('Persistent transient error');
+            
+          case 'permanent':
+            const permError = new Error('Permanent error');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)